modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -54,15 +55,16 @@ exports.update = function (req, res) {
 exports.changeProfilePicture = function (req, res) {
   var user = req.user;
   var message = null;
+  var upload = multer({ dest: './modules/users/client/img/profile/uploads/' }).single('newProfilePicture');
```

Comment:
Supporting multiple upload destinations isn't a problem.
Let's have a configuration variable like:
```javascript
uploads: {
  profileUploadDir: './modules/users/client/img/profile/uploads/'
```
and then from the code you can just do:
```javascript
var upload = multer({ dest: config.uploads.profileUploadDir }).single('newProfilePicture');
```